drivers/hmis/app/graphql/types/hmis_schema/enums/enrollment_status.rb

```diff
@@ -0,0 +1,35 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::Enums::EnrollmentStatus < Types::BaseEnum
+    description 'Status of this enrollment'
+    graphql_name 'EnrollmentStatus'
+
+    value 'OWN_ENTRY_INCOMPLETE', 'Entry Incomplete'
+    value 'ANY_ENTRY_INCOMPLETE', 'Household Entry Incomplete'
+    value 'ACTIVE', 'Active'
+    value 'EXITED', 'Exited'
+    value 'OWN_EXIT_INCOMPLETE', 'Exit Incomplete'
+    value 'ANY_EXIT_INCOMPLETE', 'Household Exit Incomplete'
+
+    def self.from_enrollment(enrollment, user:)
+      return 'OWN_ENTRY_INCOMPLETE' if enrollment.in_progress?
+
+      return 'OWN_EXIT_INCOMPLETE' if enrollment.exit_date.nil? && enrollment.exit_assessment&.present?
+
+      household_members = Hmis::Hud::Enrollment.viewable_by(user).where(household_id: enrollment.household_id)
+      return 'ANY_ENTRY_INCOMPLETE' if household_members.count > 1 && household_members.in_progress.exists?
+      return 'ANY_EXIT_INCOMPLETE' if household_members.count > 1 && household_members.find(&:exit_assessment).present?
```

Comment:
This is going to say the exit is incomplete if one person has an exit_assessment right?  I'm unclear what the find on the household_members does.

Reply:
Oh good catch. I think this is wrong. It needs a `where(ExitDate: nil)`.

"ANY_EXIT_INCOMPLETE" means that there is _another_ household member (not this one) who has an "incomplete exit." An enrollment has a "incomplete exit" if it's unexited but an Exit assessment has been started.